From: Markus Koschany <apo@gambaru.de>
Date: Fri, 10 May 2013 00:08:14 +0200
Subject: Fix FTBFS because of Wcast-align-error-flag

Byzanz still FTBFS on sparc, armel, armhf, mips, mipsel and ia64 and
this patch is still necessary to prevent this.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 620bb26..2384482 100644
--- a/configure.ac
+++ b/configure.ac
@@ -36,7 +36,7 @@ dnl For interactive UNIX (a Sun thing)
 AC_ISC_POSIX
 
 dnl decide on error flags
-AS_COMPILER_FLAGS(ERROR_CFLAGS, "-Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter -Wold-style-definition -Wdeclaration-after-statement -Wmissing-declarations -Wmissing-prototypes -Wredundant-decls -Wmissing-noreturn -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Winline -Wformat-nonliteral -Wformat-security -Wswitch-enum -Wswitch-default -Winit-self -Wmissing-include-dirs -Wundef -Waggregate-return -Wmissing-format-attribute -Wnested-externs -Wunsafe-loop-optimizations -Wpacked -Winvalid-pch -Wsync-nand")
+AS_COMPILER_FLAGS(ERROR_CFLAGS, "-Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter -Wold-style-definition -Wdeclaration-after-statement -Wmissing-declarations -Wmissing-prototypes -Wredundant-decls -Wmissing-noreturn -Wshadow -Wpointer-arith -Wwrite-strings -Winline -Wformat-nonliteral -Wformat-security -Wswitch-enum -Wswitch-default -Winit-self -Wmissing-include-dirs -Wundef -Waggregate-return -Wmissing-format-attribute -Wnested-externs -Wunsafe-loop-optimizations -Wpacked -Winvalid-pch -Wsync-nand")
 if test "x$BYZANZ_CVS" = "xyes"
 then
   AS_COMPILER_FLAGS(ERROR_CFLAGS, "-Werror")
